### ISSUE TITLE
Bug in AutoToolsBuildEnvironment

### ISSUE THREAD

**Issue by lnmcc** [conan-io/conan#2005]
conan version : (0.28.1)  OSX: 10.13

File: configure_build_environment.py 
Line: 174

I Think all append strings missing "=" symbol

eg:  `triplet_args.append("--host %s" % (host or auto_host))` should be ` triplet_args.append("--host=%s" % (host or auto_host))`

```python
       if build is not False:  # Skipped by user
            if build or auto_build:  # User specified value or automatic
                triplet_args.append("--build %s" % (build or auto_build))

        if host is not False:   # Skipped by user
            if host or auto_host:  # User specified value or automatic
                triplet_args.append("--host %s" % (host or auto_host))

        if target is not False:  # Skipped by user
            if target or auto_target:  # User specified value or automatic
                triplet_args.append("--target %s" % (target or auto_target))
```

**Comment by lasote** [comment]
Yes, it looks like the documentation use "=" symbol. Thanks. The thing is, are you experiencing problems with this? It's working fine for me without the "=" (but will work also with the =, just to confirm that it can fail without it).
Thanks

**Comment by lnmcc** [comment]
It failed on my OSX 10.13

**Comment by lasote** [comment]
Thanks for reporting!!!, will be fixed for next release.

**Comment by memsharded** [comment]
Released in 0.29, please update and try
